src/profile-logic/process-profile.js

```diff
@@ -1059,6 +1059,8 @@ function _processThread(
   const newThread: Thread = {
     name: thread.name,
     'eTLD+1': thread['eTLD+1'],
+    isPrivateBrowsing: thread.isPrivateBrowsing,
+    userContextId: thread.userContextId,
```

Comment:
I wasn't sure, but because I'm adding it from gecko (because that was easy to do) I thought we could as well keep it here too, even if we don't use it yet.

I was thinking that in the future we could pass the list of context ids with their names in `meta`, so that we could display them nicely.